backend/inspirehep/search/api.py

```diff
@@ -24,6 +24,14 @@
 class SearchMixin(object):
     """Mixin that adds helper functions to ElasticSearch DSL classes."""
 
+    @property
+    def base_index(self):
```

Comment:
``original_index`` or just ``index_name``?

Reply:
Invenio already uses _original_index and _index_name. I didn't want to create property too existing ones. I think that base_index is quite understandable - base like without_prefix one.